specs/extras/linked-token.md

```diff
@@ -0,0 +1,206 @@
+# Linked Token Spec
+
+## Outline for Linked Token Bridge
+
+<aside>
+💡 Claim: GMP will not be production-ready, and we cannot recommend its usage, until all these problems are solved.
+
+</aside>
+
+- Contracts
+    - ERC20 token named USDCTest on Calibration Network
+    - Token Controller contract on Calibration Network
+    - Token Replica contract on subnet
+- Workflow
+    - Deposit tokens from rootnet to subnet
+        - calling the rootBridge.bridgeToken() to lock up USDC tokens and initiate an IPC cross net message to the subnet token bridge
+        - subnet token bridge receives IPC message and mints USDC proxy tokens
+    - Withdraw tokens from subnet to rootnet
+        - call subnetBridge.withdrawToken() to burn proxy USDC tokens on the subnet and initiate an IPC cross net message to the root network bridge
+
+## USDC propagation flow
+
+A high-level overview of the process is shown in the following diagrams
+
+![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/75c9b610-402a-494d-9887-8258d6cc60b5/3e024f7e-8279-4b41-bd16-034bf5f603e1/Untitled.png)
+
+![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/75c9b610-402a-494d-9887-8258d6cc60b5/75ff1b6d-3576-4754-884f-57b595e8f5ee/Untitled.png)
+
+Setup and instructions for running on calibnet
+
+1. Follow IPC guide for setting up a subnet pinned to the filecoin calibnet network
+    1. https://github.com/consensus-shipyard/ipc/blob/main/docs/ipc/quickstart-calibration.md
```

Comment:
Something is off with the numeration here, probably after copying from Notion (eg there is 3.1 and 3.b below which don't make sense). Can you go through all points and make sure they're clear/consistent?

Reply:
fixed